dot/core/service.go

```diff
@@ -308,6 +314,11 @@ func (s *Service) handleRuntimeChanges(newState *rtstorage.TrieState) error {
 		return err
 	}
 
+	ver, err := s.rt.Version()
+	if err == nil {
+		go s.notifyRuntimeUpdated(ver)
+	}
```

Comment:
if this errors that is bad, so should handle that error

Reply:
updated to return err here.